### PR TITLE
Refresh /internet-of-things/appstore

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Refresh /internet-of-things/appstore based on [copydoc](https://docs.google.com/document/d/1CKwmPDDtXOIgw-jSZFwzP-qLMc_rS_dUACpHQXlg0fA/edit)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- [List additional steps to QA the new features or prove the bug has been resolved]

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-6520

### CODE REVIEW

File: templates/internet-of-things/appstore.html [hunk 1]
```diff
@@ -14,147 +14,186 @@
     {% include "shared/_18-04-eol-banner.html" %}
 
     <div class="row u-equal-height u-vertically-center">
-      <div class="col-7 col-medium-4">
+      <div class="col-6">
         <h1>The IoT App Store</h1>
```
Comment (mtruj013) [inline templates/internet-of-things/appstore.html:18]:
```suggestion
        <h1>IoT App Store</h1>
```

File: templates/internet-of-things/appstore.html [hunk 2]
```diff
@@ -14,147 +14,186 @@
...
-    <div class="col-3">
-      <ul class="p-list u-no-margin--bottom">
-        <li class="p-list__item is-ticked">CI/CD integration</li>
-        <li class="p-list__item is-ticked">Management dashboard</li>
+    <div class="col-6">
+      <p>The Dedicated Snap Store has a range of features on it's dashboard, including:</p>
+      <ul>
+        <li>Snap inclusion and exclusion</li>
+        <li>Account delegation and role management</li>
+        <li>Release management for your snaps</li>
```
Comment (mtruj013) [inline templates/internet-of-things/appstore.html:80]:
```suggestion
        <li>Software release management</li>
```

File: templates/internet-of-things/appstore.html [hunk 3]
```diff
@@ -14,147 +14,186 @@
...
-        <li class="p-list__item is-ticked">Usage analytics</li>
-        <li class="p-list__item is-ticked">Multi-tenancy</li>
-      </ul>
+<section class="p-strip">
+  <div class="row">
+    <div class="col-6">
+      <h2>Learn more about <br>the Snap Store</h2>
+    </div>
+    <div class="col-6">
+      <h3 class="p-heading--5 u-no-margin--bottom">OTA Updates</h3>
```
Comment (mtruj013) [inline templates/internet-of-things/appstore.html:95]:
```suggestion
      <h3 class="p-heading--5 u-no-margin--bottom">Over-the-air (OTA) updates</h3>
```

File: templates/internet-of-things/appstore.html [hunk 4]
```diff
@@ -14,147 +14,186 @@
...
-<section class="p-strip">
+<section class="p-strip--light">
   <div class="u-fixed-width">
-    <h2 class="p-muted-heading u-sv3">Read about the companies using app stores for IoT</h2>
-    <div class="p-logo-section has-misaligned-images">
-      <div class="p-logo-section__items">
+    <div class="p-logo-section">
+      <h2 class="p-text--small-caps">Learn about the companies using app stores</h2>
+      <p class="p-text--small">Click each logo to read a case study.</p>
+      <div class="p-logo-section__items u-no-margin--bottom">
```
Comment (mtruj013) [inline templates/internet-of-things/appstore.html:116]: Requested proper documentation on the doc for this copy, not at all clear what the final version is supposed to be

File: templates/internet-of-things/appstore.html [hunk 5]
```diff
@@ -14,147 +14,186 @@
...
-                height="71",
-                width="113",
-                hi_def=True,
-                attrs={"class": "p-logo-section__logo"},
-                loading="lazy",
-              ) | safe
+          <a href="/dell">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/87bef30b-dell-emc-logo.png",
+              alt="",
```
Comment (mtruj013) [inline templates/internet-of-things/appstore.html:121]: Missing alt text for these

File: templates/internet-of-things/appstore.html [hunk 6]
```diff
@@ -321,63 +268,40 @@ <h3 class="p-card__title">Commercial support</h3>
...
-      <a class="p-button--positive u-no-margin--bottom" href="/pricing/devices">Find out more about App Store pricing</a>
+      <a href="/pricing/devices">Learn more more about App Store pricing&nbsp;&rsaquo;</a>
     </div>
 
     <div class="col-6 p-card">
-      <h3 class="p-card__title">Snap Store Proxy</h3>
+      <h3 class="p-card__title">IoT Professional Services</h3>
       <hr class="u-sv1">
-      <p class="p-card__content">The Snap Store Proxy The Snap Store Proxy is an add-on to the IoT App Store, with a number of applications to meet your project requirements.</p>
+      <p class="p-card__content">Canonical offers an expert engineering team specialising in field deployments. We offer a range of services for all embedded projects, including:</p>
```
Comment (mtruj013) [inline templates/internet-of-things/appstore.html:278]:
```suggestion
      <p class="p-card__content">Canonical offers an expert engineering team specialising in field deployments. Explore our range of services for all embedded projects, including:</p>
```

File: templates/internet-of-things/appstore.html [hunk 7]
```diff
@@ -321,63 +268,40 @@ <h3 class="p-card__title">Commercial support</h3>
         <li class="p-list__item is-ticked">Access to our world-class technical team</li>
         <li class="p-list__item is-ticked">Private software repositories</li>
         <li class="p-list__item is-ticked">Guaranteed levels of service </li>
-        <li class="p-list__item is-ticked">Own app ecosystem control and management</li>
-        <li class="p-list__item is-ticked">Snap Store Proxy functionality</li>
       </ul>
-      <a class="p-button--positive u-no-margin--bottom" href="/pricing/devices">Find out more about App Store pricing</a>
+      <a href="/pricing/devices">Learn more more about App Store pricing&nbsp;&rsaquo;</a>
```
Comment (mtruj013) [inline templates/internet-of-things/appstore.html:272]:
```suggestion
      <a href="/pricing/devices">Learn more about App Store pricing&nbsp;&rsaquo;</a>
```

File: templates/internet-of-things/appstore.html [hunk 8]
```diff
@@ -321,63 +268,40 @@ <h3 class="p-card__title">Commercial support</h3>
...
-
-<section class="p-strip">
   <div class="row u-equal-height">
-    <div class="col-7">
+    <div class="col-6">
       <h2>Ready to innovate? </h2>
-      <p>Whether you are a startup bringing your concept to market or already established, we have considerable expertise and infrastructure to launch and support your product journey. Tell us your story and let’s work together.</p>
+    </div>
+    <div class="col-6">
+      <p class="p-heading--5">Contact us</p>
```
Comment (mtruj013) [inline templates/internet-of-things/appstore.html:299]:
```suggestion

```
Removed on the doc

**Review by mtruj013 — APPROVED** [review]
Thanks @petesfrench, LGTM! Just a few copy changes

**Comment by Sophie-32** [comment]
Apologies for the late edit, I added the content for the 'trusted by section'. The logos should be in a row, with the linked text beneath them. The logos are currently linked, use those links for the text below each logo.

**Pull request merged**